app/resume/page.tsx

```
@@ -0,0 +1,158 @@
+        <div>contact@richkevan.com</div>
+        <div>https://richkevan.com</div>
+      </div>
+      <div className="pt-4 pb-5">
+        <h3 className="text-center font-black text-2xl">Summary</h3>
+        <p className="">A highly experienced senior full stack engineer, with 5 years of experience in development with JavaScript, TypeScript, React Python, and Django. I led the efforts in creating the SSO process across multiple applications, directly improving the user experience for 50,000 active users. Designed, architected, and delivered ground up refactor of key software components reducing downtime by 70% and wait times by over 50%. </p>
+      </div>
+      <div className="pt-2 pb-5">
+        <h3 className="font-black text-2xl border-b-8 border-double border-white">Skills</h3>
+        <div className="grid grid-cols-3">
+        {resume_skills.map((skill_list) =>
+          <ul>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)